pkgs/development/python-modules/mdformat-myst/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  flit-core,
+  mdformat,
+  mdit-py-plugins,
+  pytestCheckHook,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "mdformat-myst";
+  version = "0.1.5";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "executablebooks";
+    repo = "mdformat-myst";
+    rev = "v${version}";
+    hash = "sha256-soVVTQsK3axnqfmIphvz8JZCWTTyTinUtZ7I43Kte9s=";
+  };
+
+  build-system = [ flit-core ];
+
+  dependencies = [ mdformat ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ "mdformat_myst" ];
+
+  meta = with lib; {
+    description = "Mdformat plugin for MyST compatibility";
+    homepage = "https://github.com/executablebooks/mdformat-myst";
+    license = licenses.mit;
+    maintainers = with maintainers; [ MattKang ];
```

Comment:
You need to add yourself to `maintainers/maintainer-list.nix`.

Reply:
Thanks! I believe I've fixed this in b1eab58

Reply:
Please format your commit message like other ones adding maintainers and swap the order of your commits.